packages/bot-web-ui/src/stores/transactions-store.js

```diff
@@ -174,35 +174,38 @@ export default class TransactionsStore {
 
     recoverPendingContractsById(contract_id) {
         const { ws } = this.root_store;
+        const token = this.root_store.core.client.accounts[this.root_store.core.client.loginid].token;
 
-        ws.authorized.subscribeProposalOpenContract(contract_id, response => {
-            if (!response.error) {
-                const { proposal_open_contract } = response;
+        ws.send(JSON.stringify({ authorize: token })).then(() => {
```

Comment:
@maryiafrantsava-binary Why should we authorize here specifically?

Reply:
@balakrishna-binary request `proposal_open_contract` required `authorization` of user. When the user runs the strategy `timely_balance.xml`, then goes to reports, after it returns back to DBot for any reason, I guess, the WS connection becomes closed. This line:
`ws.authorized.subscribeProposalOpenContract(contract_id, response => {`
doesn’t work with the case described above, in all other cases this line works fine. 
if you compare this situation with the **staging** and with the **production** you will see that the production calls query `authorization` and query `proposal_open_contract`, but staging runs nothing. P.S. this issue is on the **staging** only.

Reply:
This is fine for now. Please refactor the code later.

Reply:
sure